Tests/IntegrationTests/Utilities.swift

```diff
@@ -12,7 +12,7 @@ extension PostgresConnection {
     static func address() throws -> SocketAddress {
         try .makeAddressResolvingHost(env("POSTGRES_HOSTNAME") ?? "localhost", port: env("POSTGRES_PORT").flatMap(Int.init(_:)) ?? 5432)
     }
-    
+
```

Comment:
all those changes can be reverted, can't they?

Reply:
Not impossible but harder than it looks 😅
I'll do.

Reply:
Done